quelpa.el

```diff
@@ -472,6 +479,8 @@ the global var `quelpa-upgrade-p' is set to nil."
   (when (quelpa-setup-p)
     ;; shadow `quelpa-upgrade-p' taking the default from the global var
     (let* ((quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p))
+           ;; shadow `quelpa-stable-p'
+           (quelpa-stable-p quelpa-stable-p)
```

Comment:
the let binding makes it so that the globale variable is not overwritten by `setq` in the nested code, the default is taken from the global variable and as the plist is parsed it does not overwrite the global variable. I think that's kinda neat so you can do everything with one variable but in different scopes